tests/e2e_tests/cypress/integration/layoutAssertions.spec.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="Cypress" />
+
+import jwtDecode from 'jwt-decode'
+
+context('Layout assertions', () => {
+  beforeEach(() => {
+    cy.visit('/')
```

Comment:
I think it is nicer as in the previous spec file: `cy.visit(Cypress.config().baseUrl)`